script/Ciel/Day 13/13_01_02_CIEL13_6_2.txt

```diff
@@ -81,67 +95,90 @@ I don't have any obligation to answer her.
 [sha:39565659b4e9397816de1ac549875873a612b821]{
 -- Page 188, Offset 35813.
 -- もう二度とあんな機会が訪れないだけだ。
-Simply put, I'll never get a chance like that again.
+But I know I'll never get a chance like that again.
 }
 [sha:6db15e916350938bed4913e47e8db6ad4f28504e]{
 -- Page 189, Offset 35814.
 -- 「待って。待ってよ。待ちなさいよ。どういうコト？　情け？　情けをかけやがったの？　ピンに刺されるだけの虫のクセに、こんなに強くなったわたしを、まだ下に見るっていうの!?」
-"Wait. Hold on. Just hold on a minute. Why did you do that? Pity? Are you taking pity on me? You're just a bug that's stuck on a pin, and you're looking down on me even though I'm this powerful?!"
+// This supposed to be like an entomology pin, or one of her spires?
+"Hold on.
+ Wait a second.
+ Wait just one fucking second!
+ Why did you do that?
+ Pity?
+ Are you seriously taking pity on %{i}me%{/i}?
+ You're nothing but an insect that's been impaled on a pin,
+ are you seriously looking down on me even after I've become this strong?!"
 }
 [sha:ba3b672f5be7d5485ebcc2bb01eecf1a55fe23d5]{
 -- Page 10007, Offset 35815.
 -- 「そ……そうよね。そんなワケ、ないわよね。
-"R-right. There's no way that'd ever happen.
+// Secret "be nice to Noel" page
+"N-No, of course I didn't.
+ There's no way that would happen, right?
 }
 [sha:570b763ad076f066409528497373b102f25bfa70]{
 -- Page 10007, Offset 35816.
 -- だってわたしの方がずっと強いんだから。吸血鬼としてもわたしの方が上なんだから。キ、キミなんかに、怯えてなんかないんだから！」
-I'm way stronger than you, after all. Even though you're a vampire, I'm still stronger than you. I-I'd never be afraid of someone like you!"
+I mean, I'm way stronger than you are.
+ Even if you're a vampire too, I'm still stronger!
+ I-I'd never be afraid of someone like you!"
 }
 [sha:295e5a9266cd5531c0936a32dff58392e17a097d]{
 -- Page 10008, Offset 35817.
 -- 吸血鬼は<喚|わめ>きながら柱の上まで飛びあがると、忌まわしい虫を見るように俺を睨んだ。
-She flies to the top of the pillar as she shouts, before scowling at me like I'm some sort of disgusting insect.
+Still yelling, she leaps back up to the top of the pillar,
+ and then scowls back down at me like I'm some sort of disgusting insect.
 }
 [sha:038f259e09ede66eaa711bd711ac0c68b2f9f1df]{
 -- Page 192, Offset 35818.
 -- 「もういい、遊びは終わりよ。そこでみじめな肉になっちゃえ」
-"Well whatever, playtime's over. Now I'll turn you into a miserable little lump of flesh."
+// Y'know, "Become miserable meat!" just doesn't have that ring to it in EN
+"That's it, playtime's over.
+ Enjoy getting torn to shreds!"
```

Comment:
Somehow "getting torn to shreds" just sounds so generic to me that I dislike it. It's not exactly bad, but it goes against my aesthetics, somehow.

Maybe bring back mincemeat LMAO
>I'll make sad little mincemeat out of you!